website/docs/cloud-docs/api-docs/data-retention-policies.mdx

```diff
@@ -0,0 +1,127 @@
+---
+page_title: Data Retention Policies - API Docs - Terraform Cloud
+description: >-
+  Use the `/relationships/data-retention-policy` endpoints to modify the data retention policies set on different resources. Learn about the different types of data retention policies.
```

Comment:
> Is the "relationships" part of the path necessary? I took it as a general stand-in for the scope of the resource.

Im not sure the best way to represent it...there is no root `/data-retention-policy` or `/relationships/data-retention-policy` APIs. It is always `/<resource path>/data-retention-policy`...which in most cases looks like `/<resource_name>/<resource_id>/relationships/data-retention-policy`. 
So if you want to set a policy on an organization, it is `/organizations/org_name/relationships/data-retention-policy`.

Maybe the best way to generalize it would be just to say "Learn how to use the `data-retention-policy` endpoints..." with no leading `/`, so we dont imply it is a root endpoint? 

> Just wanted to capture "how long Terraform stores data" 

@trujillo-adam should we be using "Terraform Enterprise" instead of "Terraform", because it is a feature of TFE not of Terraform itself? Or do we refer to the whole collective product as Terraform?

Reply:
I could go either way with how we represent the endpoint in the meta description. In the Consul docs, we always added the leading `/` because we felt that it actually communicated a path in front of the resource and not the root.  The important thing is to include `data-retention-policy` as a keyword. I wasn't sure if `relationships` was something people would use in their searches and because engines only count the first 180 chars, I thought it would be more efficient to take it out. Up to you.

Regarding "Terraform Enterprise" vs "Terraform", aren't we going to move this information to the ptfe-releases repo? If so, then I think it would be clear from the context that this is a Terraform Enterprise API.